Etiquette/CODE_REVIEW.md

```diff
@@ -59,7 +59,7 @@ Reviewing a Pull Request
 * Do not try to make everything perfect (it never will be and otherwise we'll never deliver anything), but do not compromise on quality
 * Do not comment if you don't have any suggestions to address your comment
 * Use GitHub suggestions as much as possible
-* If you feel strong against the change - request changes and explain why you are doing that. Do not request changes for typos, missing documentation comments or code style violations, trust your colleagues to address them.
+* If you feel strongly against the change - please use `Request changes` and explain why you are doing that. **Any comment left in the PR is taken as a suggestion, and it's up to the author's discretion to accept them or not, except if you use `Request changes`.** Not every comment has to be addressed, but every comment has to be acknowledged.
```

Comment:
I'm not going to approve this in this formulation as it means that any comment that is not followed by request for changes can be ignored (equals acknowledged). It also must not be in bold as it is definitely not taking precedence of anything else in the doc. And the rest of the doc pretty clearly touches on how comments should be done and how they should be addressed. This change on the other hand makes etiquette worse as it justifies ignoring any comment author disagrees with and will only lead to more requests for changes and more frustration in the team instead of normal and healthy discussions (like one that was used to illustrate the issue this suggestion tries to address in Slack).

Reply:
I am closing this PR. You clearly have strong opinions about what you want with this piece. 👍 You can take the initiative an open a new PR with your ideas.